modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -1,25 +1,47 @@
 # frozen_string_literal: true
 
-require 'evss/error_middleware'
-
 module ClaimsApi
   module V2
     module Veterans
       class ClaimsController < ClaimsApi::V2::ApplicationController
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-        skip_before_action :authenticate, only: %i[index]
-        ICN_FOR_TEST_USER = '1012667145V762142'
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-
         def index
-          raise ::Common::Exceptions::Unauthorized if request.headers['Authorization'].blank?
-          unless params[:veteranId] == ICN_FOR_TEST_USER
-            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Resource not found')
-          end
+          veteran = build_veteran(veteran_id: params[:veteranId], loa: @current_user.loa)
+          detail  = 'Veteran not found'
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: detail) unless veteran_found?(veteran: veteran)
+
+          service   = bgs_service(veteran_participant_id: veteran.participant_id)
+          params    = { participant_id: veteran.participant_id }
+          response  = service.benefit_claims.find_claims_details_by_participant_id(params)
+          claims    = transform_response(response: response)
```

Comment:
it looks like this just responds with established claims from upstream? i know the ticket AC was a little back and forth but we ended with wanting to return both established upstream claims and our pending claims here

Reply:
Ok.  It'll take me some time to work through this one.
I'll update this comment when I've implemented that additional logic.